### PR TITLE
Show detect rule for anomaly

### PR DESCRIPTION
**Pull request by xiaohui-sun**
image

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js [hunk 1]
```diff
@@ -192,6 +192,13 @@ export default Component.extend({
    */
   metricGranularity: reads('anomaly.attributes.metricGranularity.firstObject'),
 
+  /**
+   * Anomaly detect rule
+   * @type {string}
+   */
+  detectRule: reads('anomaly.attributes.detectorComponentName'),
```
Comment (harleyjj) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: What if it only shows the rule name?  For example, in the screenshot it shows "detector_rule_1:MIGRATED_ALGORITHM", but instead show just "detector_rule_1".  This is how we display the rule in Preview and Alert Overview.

Comment (harleyjj) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: `detectRule: computed('anomaly.attributes.detectorComponentName', function () {
    return (get(this, 'anomaly').attributes.detectorComponentName || 'N/A:').split(':')[0];
  }),`

Comment (harleyjj) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: This will grab the name only or, if it's not present, return 'N/A'

Comment (xiaohui-sun) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: Still doesn't work with split function. @harleyjj

Comment (xiaohui-sun) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js]: Got it fixed finally.

**Comment by xiaohui-sun** [comment]
Close this pull and put it in https://github.com/apache/incubator-pinot/pull/4526

**Pull request closed without merging**